test/src/test/java/org/corfudb/integration/LogReplicationRoutingQueueIT.java

```diff
@@ -288,6 +306,177 @@ public void testRoutingQueueReplicationWithScopedTxn() throws Exception {
         }
     }
 
+    /**
+     * Test the order of the records on the SINK side using the entryList operation.
```

Comment:
it will be good to also add that the writes are interleaved in the test but finally verified that they are re-ordered correctly on the Sink